intel-retail/automated-self-checkout
# docs: build mkdocs

## PR Checklist



## What are you changing?
added mkdoc build files to docs folder

## Issue this PR will close

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable
check if docs are updated https://intel-retail.github.io/automated-self-checkout/index.html

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
